src/components/TodoContainer.js

```diff
@@ -1,98 +1,54 @@
-import axios from 'axios'
 import React, { useState, useEffect } from 'react'
 
 import TodosList from './TodoList'
 import Header from './Header'
 import InputTodo from './InputTodo'
 
+import { getTodosFromApi, deleteFromApi, addTodoToApi } from '../services/api'
+
 export const TodoContainer = () => {
-  const [state, setState] = useState({
-    data: {
-      todos: [],
-    },
-  })
+  const [todos, setTodos] = useState([{ id: 0, title: '', completed: false }])
```

Comment:
you can set an initial state, this is not required.
Important thing here is to keep state as flat as possible since it doesn't clone deep(ro) the tings.